assets/common.css

```
@@ -64,53 +69,71 @@ body {
   color: var(--text-color);
   scroll-behavior: smooth;
   scroll-padding-top: 60px;
+  margin: 0;
 }
 
 /* Header */
 header {
   background-color: var(--lawnchair-background-color);
-  padding: 10px;
+  padding: 14px;
   position: sticky;
   top: 0;
-  height: 50px;
+  height: 56px;
   display: flex;
+  align-items: center;
   transition: background 1s;
   z-index: 20;
+  color: white;
 }
 
 header a {
   color: white;
   text-decoration: none;
 }
 
-header > a img {
+header>a img {
   border-radius: 50%;
 }
 
-header > a {
+header>a {
   display: flex;
   align-items: center;
   gap: 10px;
   font-weight: 700;
+
+  transition: background 0.4s ease;
+  background-color: hsla(149, 75%, 51%, 0);
+  padding: 6px 0;
+  border-radius: 24px;
+}
+
+header>a:hover {
+  transition: background 0.4s ease;
+  background-color: hsla(149, 75%, 51%, 0.4);
+}
+
+.lawnchair__logo img {
+  height: 30px;
+  width: 30px;
 }
 
-header > a:hover {
-  border-bottom: 1px solid var(--accent-dark);
+.lawnchair__logo span {
+  padding-right: 14px;
 }
 
 header nav {
   margin-left: auto;
   display: flex;
   align-items: center;
-  margin-right: 10px;
+  /* margin-right: 10px; */
   gap: 5px;
 }
 
 header nav a {
   transition: all 0.2s;
   border-radius: 20px;
-  padding: 10px;
-  border: 1px solid #00000000;
+  padding: 8px 16px;
+  border: 1px solid hsla(0, 0%, 0%, 0);
```

`transparent` can work:
```suggestion
  border: 1px solid transparent;
```